### ISSUE TITLE
Page and form not resulting 

### ISSUE THREAD

**Issue by abrahamalen** [luizdepra/hugo-coder#661]
I have added an `Archives` page to the website and it's not resulting in the output. 
Any idea of the issue? 
![image](https://user-images.githubusercontent.com/53680255/156000226-94790b8e-1786-4566-9476-a2a50c8f75a2.png)  

and the subscription form I added also is not resulting in the `Subscribe` page. A form to enter email id and subscribe.
![image](https://user-images.githubusercontent.com/53680255/156000465-6626d91e-6d19-4d97-8bd6-8d7554aa28c9.png)

**Comment by luizdepra** [comment]
If I understood correctly, you will need to create an archetype template and set the type/layout in the front matter of the desired page.
I recommend you to take a look at the Hugo's Documentation and Forum. This kind of modification is beyond my current knowledge.

**Comment by abrahamalen** [comment]
Thank you @luizdepra

**Comment by abrahamalen** [comment]
Is there a code base for `archives` page in `hugo-coder` theme @luizdepra

**Comment by abrahamalen** [comment]
hey 
@luizdepra 

After I hosted my website on a custom domain, now my images are not resulting.

**Comment by luizdepra** [comment]
I don't think so. But, what is the difference from other kinds of pages? There is any other theme that has this kind of content?

**Comment by abrahamalen** [comment]
I have made the code and all but the `archives` page content is not coming up and even my` subscription form`
![image](https://user-images.githubusercontent.com/53680255/156773054-3fd406b2-bb8a-4340-906d-391fb771996b.png)
`Archives` page is coming but the content is not loading.. any idea?

I haven't committed the code, running it in local
[website](https://alenabraham.me)

**Comment by abrahamalen** [comment]
@luizdepra any ideas on the problems?
